helm_deploy/hmpps-manage-custody-mailbox-register-api/Chart.yaml

```diff
@@ -10,7 +10,7 @@ version: 1.0.0
 
 dependencies:
   - name: generic-service
-    version: 3.10.0
+    version: 3.10.3
```

Comment:
think it's better to pin at minor rather than try to keep update with patch versions?

Reply:
I've seen a few botched helm chart minor releases in the past, don't want to risk it 😄 I used to have it at pessimistic locking, but increasingly feeling more safe having it pined to minors just in case, not a big deal bumping it from time to time.